next/components/forms/steps/Summary/SummaryFormLegalText.tsx

```diff
@@ -17,7 +17,7 @@ const SummaryFormLegalText = () => {
         content={termsAndConditions}
         // TODO: Replace variant statusBar
         variant="statusBar"
-        className="rounded-10 bg-gray-50 p-4 md:p-6 lg:p-8"
+        className="rounded-[10px] bg-gray-50 p-4 md:p-6 lg:p-8"
```

Comment:
Please doublecheck, `rounded-10` should  be equal to 40px, not 10px

Reply:
Not in the old Tailwind config
https://github.com/bratislava/konto.bratislava.sk/blob/83b53a2d9a18f86045284ce5371cffa3323253bb/next/tailwind.config.js#L208